src/sdk/bucketspace_fs.js

```diff
@@ -933,6 +987,25 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
 
         return storage_classes;
     }
+
+    static async #_is_bucket_empty(name, params, ns, object_sdk) {
+        params = params || {};
+
+        let list;
+        try {
+            if (ns._is_versioning_disabled()) {
```

Comment:
`versioning` is a property of the bucket, I think you can check if the bucket versioning is disabled in a simpler way.
https://github.com/noobaa/noobaa-core/blob/779c0bf9d3c8e7a30c6d192a59924238a24bde67/src/server/system_services/schemas/nsfs_bucket_schema.js#L48-L49

Reply:
I wanted to keep the check consistent with the rest of `bucketspace_fs`.